lmdeploy/pytorch/engine/engine_instance.py

```diff
@@ -402,6 +404,10 @@ def decode(self,
         Args:
             input_ids (numpy.ndarray): the batch of input token ids
             steps (List[int]): the offset of the k/v cache
+            input_embeddings (List[List[Union[torch.Tensor, np.ndarray]]]):
+                embeddings features
+            input_embedding_ranges: (List[List[Tuple[int, int]]]):
+                the begin/end offsets of input_embeddings to input_ids
```

Comment:
`input_embeddings` and `input_embedding_ranges` are not used to pass to messages